inputs/InputData_Species.csv

```diff
@@ -1,31 +1,31 @@
-2,specific,bogr,8,1,35,2,2,2,3,1,y,0.474,0.9,0.05,0.605,15.096,0.01
-2,specific,chvi,9,1,17,2,2,2,0,2,n,0.2,0.9,0.15,0.986,70.726,0.01
-2,specific,oppo,10,1,30,4,4,1,3,1,y,0.289,0.9,0.1,2.25,15,0.01
-"1,2",fixed2,artr,1,1,150,4,3,2,0,2,n,0.02,0.9,0.2,2,400,0.01
-"1,2",fixed2,cryp,2,1,1,2,3,5,0,2,n,0.947,0.9,0.2,0.035,4.7,0
-"1,2",fixed2,chen,3,1,1,2,2,5,0,1,n,0.947,0.9,0.2,0.035,4.429,0
-"1,2",fixed2,phho,4,1,15,2,3,6,0,2,n,0.426,0.9,0.2,0.035,5,0.01
-"1,2",fixed2,arfr,5,1,15,2,3,4,0,2,n,0.426,0.9,0.2,0.035,7.48,0.01
-"1,2",fixed2,brte,6,1,1,2,3,20,0,2,n,0.947,0.9,0.15,0.02,5,0.01
-"1,2",fixed2,pssp,7,1,35,2,3,2,3,2,y,0.474,0.9,0.25,0.5,15.096,0.01
-"1,2",fixed2,bogr,8,1,35,2,2,2,3,1,y,0.474,0.9,0.05,0.605,15.096,0.01
-"1,2",fixed2,chvi,9,1,17,2,2,2,0,2,n,0.2,0.9,0.15,0.986,70.726,0.01
-"1,2",fixed2,oppo,10,1,30,4,4,1,3,1,y,0.289,0.9,0.1,2.25,15,0.01
\ No newline at end of file
+Site,treatment,name,rg,onoff,age,slow,disturb,eind,vegindv,tclass,clonal,irate,ratep,pestab,minbio,maxbio,cosurv
+all,fixed1,artr,1,1,150,15,3,5,0,2,n,0.02,0.9,0.3,2,400,0.01
+all,fixed1,cryp,2,1,1,2,3,5,0,2,n,0.947,0.9,0.2,0.035,4.7,0
+all,fixed1,chen,3,1,1,2,2,5,0,1,n,0.947,0.9,0.2,0.035,4.429,0
+all,fixed1,phho,4,1,15,2,3,5,0,2,n,0.426,0.9,0.15,0.035,5,0.01
+all,fixed1,arfr,5,1,15,2,3,5,0,2,n,0.426,0.9,0.1,0.035,7.48,0.01
+all,fixed1,brte,6,1,1,2,3,5,0,2,n,0.947,0.9,0.15,0.02,5,0.01
+all,fixed1,pssp,7,1,35,2,3,5,3,2,y,0.474,0.9,0.15,0.5,15.096,0.01
+all,fixed1,bogr,8,1,35,2,2,5,3,1,y,0.474,0.9,0.05,0.605,15.096,0.01
+all,fixed1,chvi,9,1,17,2,2,5,0,2,n,0.2,0.9,0.15,0.986,70.726,0.01
+all,fixed1,oppo,10,1,30,15,4,5,3,1,y,0.289,0.9,0.05,2.25,15,0.01
+2,specific,artr,1,1,150,15,3,2,0,2,n,0.02,0.9,0.2,2,400,0.01
+2,specific,cryp,2,1,1,2,3,5,0,2,n,0.947,0.9,0.2,0.035,4.7,0
+2,specific,chen,3,1,1,2,2,5,0,1,n,0.947,0.9,0.2,0.035,4.429,0
+2,specific,phho,4,1,15,2,3,6,0,2,n,0.426,0.9,0.2,0.035,5,0.01
+2,specific,arfr,5,1,15,2,3,4,0,2,n,0.426,0.9,0.2,0.035,7.48,0.01
+2,specific,brte,6,1,1,2,3,20,0,2,n,0.947,0.9,0.15,0.02,5,0.01
+2,specific,pssp,7,1,35,2,3,2,3,2,y,0.474,0.9,0.25,0.5,15.096,0.01
+2,specific,bogr,8,1,35,2,2,2,3,1,y,0.474,0.9,0.05,0.605,15.096,0.01
+2,specific,chvi,9,1,17,2,2,2,0,2,n,0.2,0.9,0.15,0.986,70.726,0.01
+2,specific,oppo,10,1,30,15,4,1,3,1,y,0.289,0.9,0.1,2.25,15,0.01
+"1,2",fixed2,artr,1,1,150,15,3,2,0,2,n,0.02,0.9,0.2,2,400,0.01
+"1,2",fixed2,cryp,2,1,1,2,3,5,0,2,n,0.947,0.9,0.2,0.035,4.7,0
+"1,2",fixed2,chen,3,1,1,2,2,5,0,1,n,0.947,0.9,0.2,0.035,4.429,0
+"1,2",fixed2,phho,4,1,15,2,3,6,0,2,n,0.426,0.9,0.2,0.035,5,0.01
+"1,2",fixed2,arfr,5,1,15,2,3,4,0,2,n,0.426,0.9,0.2,0.035,7.48,0.01
+"1,2",fixed2,brte,6,1,1,2,3,20,0,2,n,0.947,0.9,0.15,0.02,5,0.01
+"1,2",fixed2,pssp,7,1,35,2,3,2,3,2,y,0.474,0.9,0.25,0.5,15.096,0.01
+"1,2",fixed2,bogr,8,1,35,2,2,2,3,1,y,0.474,0.9,0.05,0.605,15.096,0.01
+"1,2",fixed2,chvi,9,1,17,2,2,2,0,2,n,0.2,0.9,0.15,0.986,70.726,0.01
+"1,2",fixed2,oppo,10,1,30,15,4,1,3,1,y,0.289,0.9,0.1,2.25,15,0.01
```

Comment:
@chaukap I'm not sure why this wasn't able to be resolved in the merge itself - are you using a mergetool?

Reply:
@kpalmqui My merge too was only allowing me to select lines, so if I chose the line with the correct values it would be in the wrong order, but if I chose the line with the wrong values it would be in the right order. However, I might not have looked closely enough. Do you have a suggestion for a mergetool?